docs/docs/policies.mdx

```diff
@@ -346,8 +346,56 @@ policies:
 * `enable_fallback_prediction` (default: `true`): Please see the
    [fallback documentation](fallback-handoff.mdx#handling-low-action-confidence) for
    further information.
-* `check_for_contradictions` (default: `true`): After training, the RulePolicy will
-   perform a check to make sure that there are no rules that contradict with each other, or with any stories.
+* `check_for_contradictions` (default: `true`):
+   Before training, the RulePolicy will perform a check to make sure that
+   actions always set slots and active loops that they set in other rules.
+   The following snippet is an example for an incomplete rule:
+
+   ```yaml-rasa
+   rules:
+   - rule: complete rule
+     steps:
+     - intent: do_something
+     - action: action_that_sets_some_slot
+     - slot_was_set:
+       - some_slot: "some value"
+
+   - rule: incomplete rule
+     steps:
+     - intent: do_something
+     - action: action_that_sets_some_slot
+   ```
+
+   In the second `incomplete rule`, `action_that_sets_some_slot` is expected to set
+   `some_slot` but does not do it.
+   There is several possible ways to fix this rule, for example:
+
+   - if `some_slot` should not be set, you should explicitly set it to `null`
```

Comment:
There should be some clarification that adding `- some_slot: null` doesn't mean the slot gets set to `null` by the policy

Reply:
I'm confused, how is that different from `- some_slot: some_value`? Users know that it is not set by the policy?

Reply:
Hmm, that's a good point. This solution should be good :thumbsup: